test/python/WMCore_t/Storage_t/StageOutMgr_t.py

```diff
@@ -12,7 +12,7 @@ class StageOutMgrTest(unittest.TestCase):
 
     def setUp(self):
         # shut up SiteLocalConfig
-        os.putenv('CMS_PATH', os.getcwd())
+        os.putenv('SITECONFIG_PATH', os.getcwd())
```

Comment:
Shouldn't this now use putenv to both `CMS_PATH` and `SITECONFIG_PATH` rather than replacing the former with the latter?

Reply:
I was going to actually delete this test, that is not testing anything at the moment. But for now, I leave it with both environment variables defined.